arches/app/models/models.py

```diff
@@ -1209,8 +1193,10 @@ class Meta:
 
 
 class ResourceInstance(models.Model):
-    resourceinstanceid = models.UUIDField(primary_key=True)
-    graph = models.ForeignKey(GraphModel, db_column="graphid", on_delete=models.CASCADE)
+    resourceinstanceid = models.UUIDField(primary_key=True, blank=True)
```

Comment:
blank=True lets us use built-in Django model validation without tripping over fields we expect a client to leave blank and have generated on the backend.